src/sequence-steps.js

```diff
@@ -35,6 +36,48 @@ const sequenceSteps = [
 	githubRelease
 ];
 
+const preReleaseSteps = [
+	gitFetchUpstreamMaster,
+	getCurrentVersion,
+	askPrereleaseIdentifier,
+	getFeatureBranch,
+	gitMergeUpstreamFeatureBranch,
+	gitLog,
+	previewLog,
+	askSemverJump,
+	updateLog,
+	updateVersion,
+	updateChangelog,
+	gitDiff,
+	gitAdd,
+	gitCommit,
+	gitTag,
+	gitPushUpstreamFeatureBranch,
+	npmPublish,
+	githubUpstream,
+	githubRelease
+];
+
+export function getCurrentVersion( [ git, options ] ) {
+	let packageJson = {};
+	try {
+		packageJson = utils.readJSONFile( "./package.json" );
+	} catch ( e ) {
+		utils.advise( "updateVersion" );
+	}
+	options.currentVersion = packageJson.version;
+	return Promise.resolve();
+}
+
+export function gitMergeUpstreamMaster( [ git, options ] ) {
+	options.branch = "master";
+	return gitMergeUpstreamBranch( [ git, options ] );
+}
+
+export function gitMergeUpstreamFeatureBranch( [ git, options ] ) {
```

Comment:
it is probably safe to remove this function completely and map above to the `gitMergeUpstreamBranch`. one less function to maintain.